projects/plugins/jetpack/modules/comment-likes.php

```diff
@@ -74,11 +74,6 @@ private function __construct() {
 		if ( ! Jetpack::is_module_active( 'likes' ) ) {
 			$active = Jetpack::get_active_modules();
 
-			if ( ! in_array( 'sharedaddy', $active, true ) && ! in_array( 'publicize', $active, true ) ) {
-				// we don't have a sharing page yet.
-				add_action( 'admin_menu', array( $this->settings, 'sharing_menu' ) );
```

Comment:
If we're not using that method anymore, I would recommend that we deprecate it, as well as the other methods it relies on.